test/WebSites/RazorWebSite/Views/PartialViewEngine/ViewPartialMissingSection.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿@*
+    For more information on enabling MVC for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
+*@
+@{
+    Layout = "/Views/Shared/_Layout.cshtml";
```

Comment:
Slightly prefer `Layout = null` here because the outer layout will never execute (again, unless you add a `try` block in the inner one).